sparkle/src/components/DataTable.tsx

```diff
@@ -118,8 +130,14 @@ export function DataTable<TData extends TBaseData>({
   }, [filter, filterColumn]);
 
   return (
-    <div className="s-flex s-flex-col s-gap-2">
-      <DataTable.Root className={className}>
+    <div
+      className={classNames(
+        "s-flex s-flex-col s-gap-2",
+        className || "",
+        widthClassName || ""
+      )}
+    >
+      <DataTable.Root>
         <DataTable.Header>
           {table.getHeaderGroups().map((headerGroup) => (
             <DataTable.Row key={headerGroup.id}>
```

Comment:
We never pass `widthClassName` to `DataTable.Row`, so we could probably remove it from the `RowProps`?

Reply:
yes we do line 195

Reply:
my bad, overlooked